src/sendMessage.js

```diff
@@ -7,10 +7,15 @@ var log = require("npmlog");
 module.exports = function(mergeWithDefaults, api, ctx) {
   return function sendMessage(msg, thread_id, callback) {
     if(!callback) callback = function() {};
-    if(typeof msg !== "string") return callback({error: "Message should be of type string and not " + typeof msg + "."});
+    if(typeof msg !== "string" && typeof msg !== "object")
```

Comment:
It's seems like we are doing a lot of type checking everywhere. Shouldn't we use a library for it instead?